react-components/src/architecture/base/commands/BaseTool.ts

```diff
@@ -85,7 +85,14 @@ export abstract class BaseTool extends RenderTargetCommand {
     // Override this to clear any temporary objects in the tool, like the dragger
   }
 
-  public onHover(_event: PointerEvent): void {}
+  public onHover(_event: MouseEvent): void {
+    // Debounce version, fast. Use this for hover effects when not
```

Comment:
Is this also supposed to say `Debounce version`?

Reply:
Changed